Makefile.js

```diff
@@ -635,7 +635,28 @@ target.gensite = function(prereleaseVersion) {
         };
     }
 
-    // 4. Loop through all files in temporary directory
+    // 4. Find out which rules are recommended and/or fixable
+
+    const recommended = new Set();
+    const fixable = new Set();
+
+    const rules = require(".").linter.getRules();
+
+    Array.from(rules.keys()).forEach(ruleName => {
+        const rule = rules.get(ruleName);
+
+        if (rule.meta.docs.recommended) {
+            recommended.add(ruleName);
+        }
+        if (rule.meta.fixable) {
+            fixable.add(ruleName);
+        }
+    });
+
+    const RECOMMENDED_TEXT = "\n\n(recommended) The `\"extends\": \"eslint:recommended\"` property in a configuration file enables this rule.";
```

Comment:
I wonder if we should just append HTML directly, without having to use replace later? We currently replace `(recommended)` and `(fixable)` with markup in Jekyll. @eslint/eslint-team thoughts?